### DISCUSSION TITLE
defog chromebook question

### DISCUSSION THREAD

**Discussion by xoidex** [3kh0/ext-remover#932] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
if i [defog](https://fog.gay) my chromebook with the 112 patch will it unenroll?
ty

**Comment by AshtonDavies** [comment, marked as answer]
No, the way E-Halcyon works is that it boots off your USB drive from the flash recovery screen by performing the E exploit, and then the Halcyon menu can install a modified v107 image and "corrupt" the operating system, which then the menu lets it boot the unenrolled environment, but the built-in recovery does not anymore until the recovery image is fixed, making you have to jumpstart with the USB or SD upon every restart.

**Reply by CGRmags** [reply]
@xoidex what is defog?

**Reply by AshtonDavies** [reply]
> @xoidex what is defog?
> 

Bypassing "The Fog" - Google's patch for Sh1mmer.

**Reply by xoidex** [reply]
thanks
